crates/iota-json-rpc-api/src/governance.rs

```diff
@@ -53,7 +55,13 @@ pub trait GovernanceReadApi {
 
     /// Return the latest IOTA system state object on-chain.
     #[method(name = "getLatestIotaSystemState")]
-    async fn get_latest_iota_system_state(&self) -> RpcResult<IotaSystemStateSummary>;
+    async fn get_latest_iota_system_state(&self) -> RpcResult<IotaSystemStateSummaryV2>;
+
+    /// Return the latest IOTA system state object on-chain (version 1).
+    /// Requires the `client-target-api-version` header to be set into
+    /// a value `<= 0.10` during requests.
+    #[method(name = "getLatestIotaSystemState", version <= "0.10")]
```

Comment:
Should we maybe go for `version < "0.11"` instead? Since this change is being introduce with `0.11.0` it might be easier to understand in the future.

Reply:
`<` is not supported I am afraid. Only `<=` and `=`.

Reply:
Worked around it as discussed: https://github.com/iotaledger/iota/pull/5449/commits/f0600e27b3486d18d05f2f44eef2d8ed6aafdee7